run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -2258,63 +2258,235 @@ Warnings:                    1
+0 CEDS_ALD2_IND   $ROOT/CEDS/v2021-06/$YYYY/ALD2-em-anthro_CMIP_CEDS_$YYYY.nc          ALD2_ind          1750-2019/1-12/1/0 C xy kg/m2/s ALD2  26    1 5
+0 CEDS_ALD2_TRA   $ROOT/CEDS/v2021-06/$YYYY/ALD2-em-anthro_CMIP_CEDS_$YYYY.nc          ALD2_tra          1750-2019/1-12/1/0 C xy kg/m2/s ALD2  26    1 5
+0 CEDS_ALD2_RCO   $ROOT/CEDS/v2021-06/$YYYY/ALD2-em-anthro_CMIP_CEDS_$YYYY.nc          ALD2_rco          1750-2019/1-12/1/0 C xy kg/m2/s ALD2  26    1 5
+0 CEDS_ALD2_SLV   $ROOT/CEDS/v2021-06/$YYYY/ALD2-em-anthro_CMIP_CEDS_$YYYY.nc          ALD2_slv          1750-2019/1-12/1/0 C xy kg/m2/s ALD2  26    1 5
+0 CEDS_ALD2_WST   $ROOT/CEDS/v2021-06/$YYYY/ALD2-em-anthro_CMIP_CEDS_$YYYY.nc          ALD2_wst          1750-2019/1-12/1/0 C xy kg/m2/s ALD2  26    1 5
+0 CEDS_MEK_AGR    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_agr           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_MEK_ENE    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_ene           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_MEK_IND    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_ind           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_MEK_TRA    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_tra           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_MEK_RCO    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_rco           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_MEK_SLV    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_slv           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_MEK_WST    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_wst           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26    1 5
+0 CEDS_HCOOH_AGR  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_agr         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
+0 CEDS_HCOOH_ENE  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_ene         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
+0 CEDS_HCOOH_IND  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_ind         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
+0 CEDS_HCOOH_TRA  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_tra         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
+0 CEDS_HCOOH_RCO  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_rco         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
+0 CEDS_HCOOH_SLV  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_slv         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
+0 CEDS_HCOOH_WST  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_wst         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26    1 5
```

Comment:
Hierarchy should be 4 here. See comment in GCClassic version of this file.

Reply:
Now fixed in 3f69125

https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem#L2362-L2489